Update parsing.md to move data as the last option (#8655)
DataDog/documentation: content/en/logs/processing/parsing.md

| `hostname`                                      | Matches a hostname.                                                                                                                |
| `ipOrHost`                                      | Matches a hostname or IP.                                                                                                          |
| `port`                                          | Matches a port number.                                                                                                             |

> | `data`                                          | Matches any string including spaces and newlines. Equivalent to `.*`. Use when none of above matchers are appropriate.                                                              |

Reviewer: ```suggestion
| `data`                                          | Matches any string including spaces and newlines. Equivalent to `.*` in regex. Use when none of above patterns is appropriate.                                                              |
```
